bin/_k3d-config.yml

```diff
@@ -0,0 +1,3 @@
+apiVersion: k3d.io/v1alpha4
+kind: Simple
+image: rancher/k3s:v1.27.6-k3s1
```

Comment:
As a favor to our future selves, can we rename this file so that it clearly indicates what we're using it for? This isn't a general purpose k3d config.

What about something like `test/integration/deep/calico-k3d.yml`?

And then in the file we should include a comment that references the k3x issue we're tracking.

Reply:
Sounds good. The k3d issue was already referred to on the calling side, but I can add it to the config file as well.